code/modules/mob/living/simple_animal/slime/life.dm

```diff
@@ -129,8 +134,27 @@
 	else
 		Tempstun = 0
 
+	if(stat != DEAD)
+		var/co2_percentage =0
+		if(("co2" in environment.gases) && environment.total_moles())
```

Comment:
Then I still can somehow remove all the CO2, leaving the turf with flat zero moles. Unlikely, but possible.